pkgs/tools/text/ov/default.nix

```diff
@@ -0,0 +1,35 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "ov";
+  version = "0.13.0";
+
+  src = fetchFromGitHub {
+    owner = "noborus";
+    repo = "ov";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-vBPhCSor3wGCawz+097Lw29xgW6z5fV5PAMAq7TBiNM=";
+  };
+
+  vendorHash = "sha256-y3oSL1W2cjt6oUVbglHhun3XNCidqb7LTXtoA25+mpo=";
+
+  ldflags = [
+    "-X main.Version=v${version}"
+    "-X main.Revision=${src.rev}"
+  ];
+
+  excludedPackages = [
+    "exec"
+    "search"
+    "simple"
+    "tail"
+  ];
+
+  meta = with lib; {
+    description = "Feature-rich terminal-based text viewer";
+    homepage = "https://noborus.github.io/ov";
+    license = licenses.mit;
+    platforms = platforms.linux;
```

Comment:
Is this an inherent limitation that it does not work on darwin?

Reply:
I don't have a darwin at hand to test it. I'm very sure it would work there otherwise, based on https://github.com/noborus/ov#macports-(macos)

Reply:
You don't need to have a Darwin to test, you can relax the platform constraint and let CI/reviewers do it for you.
If it's broken, we would restrain the platform or mark it as broken on that platform.

Reply:
Added darwin. Looking through the code I'm sure it will build as-is.

Reply:
(running the review, we will know soon)